src/swarm/neo/request/RequestEventDispatcher.d

```diff
@@ -582,6 +578,8 @@ public struct RequestEventDispatcher
             flags = flags.Receive;
             if ( this.waitingYielders() )
```

Comment:
It occurs to me that these `waitingForX` methods should probably be combined, so as not to iterate over the array multiple times. Can be another PR.

Reply:
Good idea, I'll follow-up with the next PR in series.